pkgs/http_client_conformance_tests/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
 
 dependencies:
   async: ^2.8.2
-  dart_style: ^2.3.7
+  dart_style: '>=2.3.7 <4.0.0'
```

Comment:
Can we just change this to ^3.0?

Reply:
Only if you want to bump the min SDK everywhere to 3.7, me thinks

Reply:
Gotcha